These ioctls are not actually supported (and will probably not be in the
close future), and are not available on Linux either, so don't expose
them to application at the risk of them complaining that they don't work
(e.g. xterm using TIOCLSET).

---
 sysdeps/mach/hurd/bits/ioctls.h |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/sysdeps/mach/hurd/bits/ioctls.h
+++ b/sysdeps/mach/hurd/bits/ioctls.h
@@ -272,11 +272,13 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_
 #define	FIOSETOWN	_IOW('f', 124, int)	/* set owner */
 #define	FIOGETOWN	_IOR('f', 123, int)	/* get owner */
 
+#if 0
 /* socket i/o controls */
 #define	SIOCSHIWAT	_IOW('s',  0, int)		/* set high watermark */
 #define	SIOCGHIWAT	_IOR('s',  1, int)		/* get high watermark */
 #define	SIOCSLOWAT	_IOW('s',  2, int)		/* set low watermark */
 #define	SIOCGLOWAT	_IOR('s',  3, int)		/* get low watermark */
+#endif
 #define	SIOCATMARK	_IOR('s',  7, int)		/* at oob mark? */
 #define	SIOCSPGRP	_IOW('s',  8, int)		/* set process group */
 #define	SIOCGPGRP	_IOR('s',  9, int)		/* get process group */
@@ -336,7 +338,9 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_
 # define OTIOCGETD	_IOR('t', 0, int)	/* get line discipline */
 # define OTIOCSETD	_IOW('t', 1, int)	/* set line discipline */
 #endif
+#if 0
 #define	TIOCHPCL	_IO('t', 2)		/* hang up on last close */
+#endif
 #define	TIOCGETP	_IOR('t', 8,struct sgttyb)/* get parameters -- gtty */
 #define	TIOCSETP	_IOW('t', 9,struct sgttyb)/* set parameters -- stty */
 #define	TIOCSETN	_IOW('t',10,struct sgttyb)/* as above, but no flushtty*/
@@ -389,6 +393,7 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_
 #define		PENDIN		0x20000000	/* tp->t_rawq needs reread */
 #define		DECCTQ		0x40000000	/* only ^Q starts after ^S */
 #define		NOFLSH		0x80000000	/* no output flush on signal */
+#if 0
 #define	TIOCLBIS	_IOW('t', 127, int)	/* bis local mode bits */
 #define	TIOCLBIC	_IOW('t', 126, int)	/* bic local mode bits */
 #define	TIOCLSET	_IOW('t', 125, int)	/* set entire local mode word */
@@ -414,6 +419,7 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_
 #define	OTTYDISC	0
 #define	NETLDISC	1
 #define	NTTYDISC	2
+#endif
 
 /* From 4.4 <sys/ttydev.h>.   */
 #ifdef USE_OLD_TTY
